hadoop-ozone/s3gateway/src/test/java/org/apache/hadoop/ozone/client/OzoneBucketStub.java

```diff
@@ -358,6 +359,18 @@ public void deleteKey(String key) throws IOException {
     keyDetails.remove(key);
   }
 
+  @Override
+  public Map<String, Pair<String, String>> deleteKeysQuiet(List<String> keyList, Boolean isQuiet) throws IOException {
+    Map<String, Pair<String, String>> keyErrorMap = new HashMap<>();
+    for (String key : keyList) {
+      if (!keyDetails.containsKey(key)) {
+        keyErrorMap.put(key, Pair.of("KEY_NOT_FOUND", "Key does not exist"));
+      }
+      keyDetails.remove(key);
```

Comment:
nit:

```suggestion
      if (!keyDetails.containsKey(key)) {
        keyErrorMap.put(key, Pair.of("KEY_NOT_FOUND", "Key does not exist"));
      } else {
        keyDetails.remove(key);
      }
```

or even:

```suggestion
      if (keyDetails.remove(key) == null) {
        keyErrorMap.put(key, Pair.of("KEY_NOT_FOUND", "Key does not exist"));
      }
```